tests/drivers/comparator/gpio_loopback/testcase.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  harness_config:
+    fixture: gpio_loopback
+  tags:
+    - drivers
+    - comparator
+tests:
+  drivers.comparator.gpio_loopback.mcux_acmp:
+    platform_allow:
+      - frdm_ke15z
+  drivers.comparator.gpio_loopback.nrf_comp:
+    extra_args:
+      - SNIPPET="gpio_loopback_nrf_comp"
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
nrf54l15dk is the only supported i.e. please remove nrf54l15pdk

Reply:
do you folks have plan to drop the board entirely?

Reply:
yep, https://github.com/zephyrproject-rtos/zephyr/pull/79944